### PR TITLE
Register btcpay rewording

### LINKED ISSUE #812: Updates to Introduction documents
- There have been a few recent questions in the chat about starting/thirdparty/where to login 
- Comment for [reference](https://chat.btcpayserver.org/btcpayserver/pl/6yabuf9raincz8gp15poh15oja)
- Possibly simplify or make it more clear

### LINKED ISSUE #840: Make third party hosts list more obvious
ThirdParyHosts.md is great and in-depth doc. We should add a tip like this

:::tip
foo
:::

To make the link more obvious.

Right now we display it here and like this:
Screenshot from 2021-02-12 11-00-39

May be a good idea to add a light tip in the intro or somewhere where it's easy to locate, I found myself browsing through the page quite often lately to try to find it.

### PR DESCRIPTION
**Pull request by Zaxounette**
Hopefully closes #812 and closes #840 

* Adds wording redirecting viewers to appropriate sections. Mainly the demo official demo instance, third party hoster list (directory) or deployment page.
* Add rewording and addition in an attempt to make more clear the fact that BTCPay is not a company.
* Add `tip` mark to make directory third party hoster list more visually seen..

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -48,7 +48,10 @@ In layman's terms, BTCPay Server is a self-hosted and automated invoicing system
 
 ### How is it different
 
-BTCPay is code, not a company. There is no third-party between a merchant and a customer. Merchant is always in full control of the funds. There are no processing or subscription fees. BTCPay Server is free to use and completely open-source, so developers or security auditors can always inspect the quality of the code.
+BTCPay Server is free, open-source, self-hostable software. 
+BTCPay Server is not a company. To use BTCPay Server, you can register an account on the [official demo](https://mainnet.demo.btcpayserver.org/Account/Login), [deploy](./Deployment.md) your own instance, or use a [third-party host](./ThirdPartyHosting.md).
```
Comment (dennisreimann) [inline docs/README.md:52]: Not sure if we should promote "use" with the demo instance. We should be more explicit here imho.

**Review by dennisreimann — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (Zaxounette) [reply docs/README.md:52]: I get what you're saying.
Yet it's stated as a demo.

Maybe the wording as follows could be better ?

```suggestion
BTCPay Server is not a company. To use BTCPay Server, [deploy](./Deployment.md) your own instance, or use a [third-party host](./ThirdPartyHosting.md).

 To explore what BTCPay Server has to offer, head over to the [official demo](https://mainnet.demo.btcpayserver.org/Account/Login).
```

Comment (dennisreimann) [reply docs/README.md:52]: I think this is better.

**Review by dennisreimann — APPROVED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -48,7 +48,12 @@ In layman's terms, BTCPay Server is a self-hosted and automated invoicing system
 
 ### How is it different
 
-BTCPay is code, not a company. There is no third-party between a merchant and a customer. Merchant is always in full control of the funds. There are no processing or subscription fees. BTCPay Server is free to use and completely open-source, so developers or security auditors can always inspect the quality of the code.
+BTCPay Server is free, open-source, self-hostable software. 
+BTCPay Server is not a company. To use BTCPay Server, [deploy](./Deployment.md) your own instance, or use a [third-party host](./ThirdPartyHosting.md).
+
+ To explore what BTCPay Server has to offer, head over to the [official demo](https://mainnet.demo.btcpayserver.org/Account/Login).
+
+While using BTCPay Server, there is no third-party between a merchant and a customer. The merchant is always in full control of their funds. There are no processing or subscription fees. BTCPay Server is free to use and completely open-source, so developers or security auditors can always inspect the quality of the code.
```
[the remark this answers is not in the document]
Comment (britttttk) [reply docs/README.md:56]: I feel like we should avoid:
- using "no third-party" because it might be confusing with our third-party host terminology.
- using merchant & customer on main intro because of people using the software for donations.

@Zaxounette what do you think about these neutral replacements?

**Review by britttttk — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Zaxounette) [reply docs/README.md:56]: Agreed, thanks ! :)
